client/verta/verta/operations/monitoring/alert/_entities/alert.py

```diff
@@ -432,13 +458,23 @@ def create(
             name=name,
             monitored_entity_id=self._monitored_entity_id,
             alerter=alerter,
-            summary_sample_query=summary_sample_query,
+            summary_sample_query=combined_query,
             notification_channels=notification_channels,
             created_at_millis=time_utils.epoch_millis(created_at),
             updated_at_millis=time_utils.epoch_millis(updated_at),
             last_evaluated_at_millis=time_utils.epoch_millis(last_evaluated_at),
         )
 
+    @property
+    def base_summary_query(self):
+        return self._base_summary_query
+
+    def _add_query_to_default_query(self, summary_sample_query):
+        summary_query = self._base_summary_query + summary_sample_query.summary_query
+        updated_summary_sample_query = copy.copy(summary_sample_query)
+        updated_summary_sample_query.summary_query = summary_query
+        return updated_summary_sample_query
```

Comment:
The method name makes it sound like it's an in-place operation rather than a method that returns a new object. It also feels a bit unintuitive to add a sample query to a summary query, but maybe that's a bigger design matter. 🤔

Reply:
I'm renaming it to `_combine_query_with_default_summary`